examples/axis-annotation/index.js

```diff
@@ -0,0 +1,61 @@
+class Annotation {
+    constructor(value, velocity) {
+        this.value = value;
+        this.velocity = velocity;
+    }
+
+    animate() {
+        this.velocity += (Math.random() - 0.5) * 0.01;
+        this.value += this.velocity;
+        if (this.value < scale.domain()[0]) {
```

Comment:
<minor/>

I find indexed values returned by accessors really hard to read `scale.domain()[0]`, perhaps pull out the scale range as a variable?
```
const range = [0, 100];
...
this.value = range[0] + (range[0] - this.value)
...
const scale = d3.scaleLinear().domain(range);
```